src/containers/Sidebar/Sidebar.test.jsx

```diff
@@ -65,12 +68,34 @@ describe('<Sidebar />', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders correctly when code management is hidden', () => {
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        ...initialState.portalConfiguration,
```

Comment:
Because `enableCodeManagementScreen` is the only key within `portalConfiguration` in the store, we probably don't need to spread on the `initialState.portalConfiguration` here (i.e., could remove this line).